app/components/agenda/agenda-header/agenda-tabs.js

```diff
@@ -41,11 +25,16 @@ export default class AgendaAgendaHeaderAgendaTabsComponent extends Component {
   get currentAgendaItemId() {
     const currentRoute = this.router.currentRoute;
     let agendaItemsRoute = currentRoute;
-    if (currentRoute && currentRoute.name.startsWith('agenda.agendaitems.agendaitem')) {
+    if (currentRoute?.name?.startsWith('agenda.agendaitems.agendaitem')) {
       while (agendaItemsRoute.name !== 'agenda.agendaitems.agendaitem') {
         agendaItemsRoute = agendaItemsRoute.parent;
       }
       return agendaItemsRoute.params.agendaitem_id;
+    } else if (currentRoute?.name?.startsWith('agenda.agendaitems')) {
+      while (agendaItemsRoute.name !== 'agenda.agendaitems') {
+        agendaItemsRoute = agendaItemsRoute.parent;
+      }
+      return agendaItemsRoute.queryParams.anchor;
```

Comment:
When going from "Overview" to "Details" and back to "Overview" via the tabs, the anchor would be reset to the first agendaitem, regardless of whether a different agendaitem was previously selected. This occurred because in the "Overview" tab we are in the `agenda.agendaitems` route. In the original code, we could never get an agendaitem's id, so the route of "Details" tab defaulted to the first agendaitem, and then coming back to "Overview" the anchor would get replaced with the id of the first agendaitem.

To fix this, we try to get the agendaitem id from the anchor query parameter if we failed to get it from the route itself.